### ISSUE TITLE
[Bug]: Build failure on Arch Linux

### ISSUE THREAD

**Issue by LumaLivy** [ciderapp/Cider#1403]
### Describe the Bug

Building `cider` or `cider-git` fails as follows (I chopped out the irrelevant parts of the terminal output with [...])

```
[...]

> cider@1.5.6 dist /home/lumalivy/.cache/yay/cider/src/Cider
> npm run build && electron-builder "dir"

[...]

Unknown argument: dir
 ELIFECYCLE  Command failed with exit code 1.
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: cider
~
```

### Steps to Reproduce

Build the project using a package manager, I used `yay` and tested both `cider` and `cider-git`

### Environment Information

- OS: Arch Linux
- App Version and/or Commit ID: `1.5.6-4` as reported by aur

### Anything else?

Not sure what to do, I'm sorry I can't investigate further. I'm guessing `"dir"` isn't supposed to be an argument as part of that build command, but an actual path instead? I can provide the full output of the build command if needed.

PS, thanks! Cider is an awesome app :)

### Code of Conduct

- [X] I agree to follow this Cider's [Code of Conduct](https://github.com/ciderapp/Cider/blob/main/CODE_OF_CONDUCT.md)

### Disclaimer

- [X] I acknowledge the [Support Disclaimer](https://docs.cider.sh/support/disclaimer)

**Comment by LumaLivy** [comment]
Okay, I looked at the PKGBUILD and found this:

```
build() {
    cd "${srcdir}/${_pkgname}"

    echo "Building ${_pkgname} on v${pkgver} : [Install Build Dependencies] | Build | Done"
    pnpm install --frozen-lockfile

    echo "Building : Install Build Dependencies | [Build] | Done"
    pnpm dist --linux dir

    echo "Building : Install Build Dependencies | Build | [Done]"
}
```

Changing `pnpm dist --linux dir` to `pnpm dist --linux --dir` seemed to change the output of the line I highlighted in the original comment (changed to `> npm run build && electron-builder`), but then I was getting complaints that `electron-builder` is not a command, so I installed it globally with yarn and then now it just fails to build

```
Building : Install Build Dependencies | [Build] | Done

> cider@1.5.6 dist /home/lumalivy/.cache/yay/cider/src/Cider/node_modules
> npm run build && electron-builder


> cider@1.5.6 build
> tsc && npm run compile-less


> cider@1.5.6 compile-less
> lessc ./src/renderer/style.less ./src/renderer/style.css

  • electron-builder  version=23.3.3 os=5.19.4-arch1-1
  ⨯ Cannot find package.json in the /home/lumalivy/.cache/yay/cider/src/Cider/node_modules  failedTask=build stackTrace=Error: Cannot find package.json in the /home/lumalivy/.cache/yay/cider/src/Cider/node_modules
    at Packager.readProjectMetadataIfTwoPackageStructureOrPrepacked (/home/lumalivy/.config/yarn/global/node_modules/app-builder-lib/src/packager.ts:410:11)
    at Packager.build (/home/lumalivy/.config/yarn/global/node_modules/app-builder-lib/src/packager.ts:328:24)
    at Object.executeFinally (/home/lumalivy/.config/yarn/global/node_modules/builder-util/src/promise.ts:12:14)
 ELIFECYCLE  Command failed with exit code 1.
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: cider
 ```
 
At this point I'm tinkering with things I don't really understand, so my apologies. I have a feeling there's something messed up with `dir` in the PKGBUILD being passed down into the electron-builder command with quotes around it incorrectly because `dir` seems to be a valid build target as seen in the instructions on https://www.electron.build/

**Comment by coredev-uk** [comment]
Try clearing your clone cache. `rm -r .cache/yay/cider-git`

**Comment by LumaLivy** [comment]
Hi there, sorry for getting back a couple days late. Unfortunately I already tried that a couple times between attempted reinstalls. Has anyone else managed to build it successfully on arch? I also tried using nodejs-lts-gallium for the node version but that didn't seem to make a difference in my case. Hoping the issue isn't related to my setup, I've had no issues in the past updating/installing cider before.

Thank you very much for the help!
